### ISSUE TITLE
Transfer Learning: 23. Lab: Cifar10 Aside improvement

### ISSUE THREAD

**Issue by brahmwindeler** [udacity/sdc-issue-reports#399]
The lab instructs us to use the classifier from the Traffic Sign project with the CIFAR10 data, instructing us how to load it and indicating that the number of output classes needs to be changed from 43 to 10. However, it neglects to mention that the label data is also in a slightly different format: nested arrays containing single scalars vs. arrays of scalars [e.g. shape (50000, 1) vs. (50000, )]. While the tf.one_hot encoding function seems to be flexible enough to figure this out, the code required to visualize the data set isn't guaranteed to do the same [e.g. trying to do np.bincount(y_train) won't work]. I would recommend mentioning that the difference in label data encoding and that it might help to be "flattened" like so: 

from keras.datasets import cifar10
(X_train, y_train), (X_test, y_test) = cifar10.load_data()
y_train = np.hstack(y_train)
y_test = np.hstack(y_test)

**Comment by domluna** [comment]
Thanks @brahmwindeler I've added code to the section that flattens the `y_train` and `y_test` array.
